src/theme/index.js

```diff
@@ -116,6 +114,11 @@ const theme = createTheme({
           border: '0',
           width: '100%',
         },
+        input: {
+          '&:-webkit-autofill': {
+            transitionDelay: '5000s',
```

Comment:
5000 seconds is very long delay. I believe we try to remove the autofill, so let's try to find another aproach to do that